docs/tutorials/acme/dns-validation.rst

```diff
@@ -86,30 +89,52 @@ Once we have created the above Issuer we can use it to obtain a certificate.
 .. code-block:: yaml
    :linenos:
 
-   apiVersion: certmanager.k8s.io/v1alpha1
-   kind: Certificate
-   metadata:
-     name: example-com
-     namespace: default
-   spec:
-     secretName: example-com-tls
-     issuerRef:
-       name: letsencrypt-staging
-     commonName: '*.example.com'
-     dnsNames:
-     - example.com
-     - foo.com
-     acme:
-       config:
-       - dns01:
-           provider: prod-dns
-         domains:
-         - '*.example.com'
-         - example.com
-       - dns01:
-           provider: cf-dns
-         domains:
-         - foo.com
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: Certificate
+    metadata:
+      creationTimestamp: 2019-08-30T04:06:43Z
+      generation: 1
+      labels:
+        app.kubernetes.io/instance: imprecise-dingo
+        app.kubernetes.io/managed-by: Tiller
+        app.kubernetes.io/name: kuard
+        app.kubernetes.io/version: "1.0"
+        helm.sh/chart: example-com-0.1.0
+      name: quickstart-example-tls
+      namespace: default
+      ownerReferences:
+      - apiVersion: extensions/v1beta1
+        blockOwnerDeletion: true
+        controller: true
+        kind: Ingress
+        name: example-com-0.1.0
+        uid: a3e9f935-ea87-11e8-82f8-42010a8a00b5
+      resourceVersion: "9295"
+      selfLink: /apis/certmanager.k8s.io/v1alpha1/namespaces/default/certificates/quickstart-example-tls
+      uid: 68d43400-ea92-11e8-82f8-42010a8a00b5
+    spec:
+      dnsNames:
+      - 'example.your-domain.com'
+      issuerRef:
+        kind: Issuer
+        name: letsencrypt-staging
+      secretName: quickstart-example-tls
+    status:
+      conditions:
+      - lastTransitionTime: 2019-08-30T04:09:47Z
+        message: Certificate is up to date and has not expired
+        reason: Ready
+        status: "True"
+        type: Ready
+      notAfter: 2019-11-28T03:09:46Z
+    events:
+      Type     Reason          Age                From          Message
+      ----     ------          ----               ----          -------
+      Normal   CreateOrder     9m                 cert-manager  Created new ACME order, attempting validation...
+      Normal   DomainVerified  8m                 cert-manager  Domain "example.your-domain.com" verified with "http-01" validation
+      Normal   IssueCert       8m                 cert-manager  Issuing certificate...
+      Normal   CertObtained    7m                 cert-manager  Obtained certificate from ACME server
+      Normal   CertIssued      7m                 cert-manager  Certificate issued Successfully
```

Comment:
Can you revert your changes to the Certificate resource here and keep the old one, simply removing the `spec.acme` stanza?

I think this may mislead users, as not all of them will contain this same metadata, plus you've changed the dnsNames listed on the Certificate too. There's also no one single command that you can run that will render the YAML followed by the Events in the fashion you've done here - I think simply keeping the old, minus `spec.acme` would be best 😄

Reply:
fixed thx